Cargo.toml

```diff
@@ -198,32 +196,33 @@ hotshot-utils = { path = "crates/hotshot/utils" }
 vid = { path = "vid", features = ["parallel", "keccak256"] }
 
 bytesize = "1.3"
-committable = "0.2"
+# committable = "0.2"
```

Comment:
```suggestion
```

Reply:
@jbearer @Ancient123 could we get this committable@0.2.5 to crates.io?